### DISCUSSION TITLE
What font was used in the Qur'an app?

### DISCUSSION THREAD

**Discussion by digitaldrreamer** [sufone/qawl-svelte#86] [Q&A] [answered]
Assalamu aleikum wa rahmatullahi wa barakaatuhu brother.
Just wanted to know what font was used in the Mushaf images. Help will be greatly appreciated. Jazakumullahu Khairan

**Comment by mr-islam** [comment]
Wa alaykum as salam wa rahmatullahi wa barakatuhu dear brother

Qawl is not using a font, rather they're images directly from the Mushaf scan, I got them from [here](https://qurancomplex.gov.sa/en/kfgqpc-quran-hafs/).

If you would like the fonts, you can find them [here](https://github.com/mustafa0x/qpc-fonts).

**Comment by Libax470** [comment]
Assalamu Alaikum Wa Rahmatullahi Wa Barakatuh my brother @mr-islam, 
Is it possible to get [this Mushaf images](https://raw.githubusercontent.com/sufone/qawl/master/public/mushaf/1.png) as a high quality images or svg. I am trying to use it for a youtube video.
JazakAllahu Khairan.

**Comment by mr-islam** [comment]
Wa alaykum as salam wa rahmatullahi wa barakatuhu dear brother @Libax470 

All the images are available [here](https://github.com/sufone/qawl/tree/master/public/mushaf)

Let me know if I can help with anything else, and I'll convert this to a discussion

**Comment by Libax470** [comment]
Yes brother, I need your help. Alhamdulillah I have those images but when I try to create a video using those images, they become blurry. I was wondering if I could find a high quality images or a font of this Mushaf with the circle around the end ayahs. I am trying to create high quality videos like [this](https://youtu.be/B5MpA7qZJEQ) or [this](https://youtu.be/V4jNx8zYUJw) if possible. Thank you for your attention.

**Comment by maaldheefee** [comment]
https://dm.qurancomplex.gov.sa/

This is the original source of those images as Illustrator vector files.

**Comment by mr-islam** [comment]
I think the original images don't have circles around the ayah numbers, right?

This might be slow, but you @Libax470 can try screenshotting from quran.com, like this:

![image](https://github.com/sufone/qawl/assets/17675428/b8c7f054-7daa-4132-acd1-140c8f167554)

It's clear and has ayah number decoration

I increased the font size in quran.com settings, and chose the King Fahad V2 font

And you can mention quran.com in the video description and give them credit

**Comment by Libax470** [comment]
The original images from quran.com and qurancomplex don't have circles around the ayah. I do have the 2005 and 2013 quran fonts. I thought a new font was used in this [Mushaf](https://raw.githubusercontent.com/sufone/qawl/master/public/mushaf/50.png) (that has circles around the ayah). It is just that I prefer it over the 2005 or 2013 fonts but I am happy to use them too.

**Reply by maaldheefee** [reply]
The newer fonts used to have some issues. Make sure to use the latest version.

![KFQPC2](https://github.com/sufone/qawl/assets/16198035/aea4dd67-112b-4007-a20f-a5b8fb0dd00e)

![KFQPC1](https://github.com/sufone/qawl/assets/16198035/d2144000-dc3e-4384-a528-d3b507496889)

The mushaf that has circles around them is the original vectorized scans of the new mushaf. From which the new fonts were made. The version with circles had low quality curves and that version was removed from the official website. I have a vector PDF of that if you need that version.

**Comment by Libax470** [comment]
Yes, please send the vector PDF.

**Reply by maaldheefee** [reply, marked as answer]
https://1drv.ms/f/s!AltYk8ze7yLqjDsh441JLJYbt93Z?e=cyyxys

**Comment by Libax470** [comment]
JazakAllahu Khairan @maaldheefee I got the pdf and the ai files now. And thank you @mr-islam, may Allah grant you guys Jannatul Firdaus.

**Reply by mr-islam** [reply]
Ameen for all of us

Thank you so much @maaldheefee, the files will be super helpful 🌹
